app/script/modules/show_section.js

```diff
@@ -0,0 +1,23 @@
+export default function showSection() {
+  const navigation = document.querySelector('.nav-links');
+  const links = navigation.getElementsByClassName('nav-link');
+
+  for (let count = 0; count < links.length; count += 1) {
+    links[count].addEventListener('click', (e) => {
+      const current = document.querySelectorAll('.active');
+      const sections = document.querySelectorAll('.content');
+      current[0].className = current[0].className.replace(' active', '');
+      e.target.className += ' active';
+
+      sections[0].className = sections[0].className.replace(' show', '');
+
+      for (let counter = 0; counter < sections.length; counter += 1) {
+        if (e.target.getAttribute('data-show') === sections[counter].id && sections[counter].classList.contains('show') !== true) {
+          sections[counter].className += ' show';
+        } else {
+          sections[counter].classList.remove('show');
+        }
+      }
+    });
+  }
+}
```

Comment:
- [ ] I think there may be an issue with your application navbar. I am unable to navigate through the pages. As can see from the URL bar I have tried going through the pages but the display does not change.

- [ ] Also your date and time from luxon is not displayed at all

  List page | Add new page | Contact page
  :---:|:---:|:---:
  ![htm-list](https://user-images.githubusercontent.com/36542986/192487978-bc1c18bd-6535-4f5d-a589-a7e56d01b936.jpg) | ![htm-add](https://user-images.githubusercontent.com/36542986/192487970-5ac6127c-07c3-4a66-91c5-45a06bb47907.jpg) | ![htm-contact](https://user-images.githubusercontent.com/36542986/192487946-e9a32e10-6d8b-43fb-9643-f8d5d732641a.jpg)

  I think these issues may have to do with this:

  ![lu](https://user-images.githubusercontent.com/36542986/192488463-c0730ae2-bb67-49ac-9ce5-060e1d2d3def.jpg)

Reply:
Thanks for the feedback. I'm going to fix that.🙃

Reply:
Hi @TedLivist, the bugs are fixed. A confirmation from your side would be appreciated.🙃